packages/wonder-blocks-data/util/response-cache.test.js

```diff
@@ -0,0 +1,346 @@
+// @flow
+import {ResponseCache} from "./response-cache.js";
+
+import type {IRequestHandler} from "./types.js";
+
+describe("./response-cache.js", () => {
+    afterEach(() => {
+        /**
+         * This is needed or the JSON.stringify mocks need to be
+         * mockImplementationOnce. This is because if the snapshots need
+         * to update, they write the inline snapshot and that appears to invoke
+         * prettier which in turn, calls JSON.stringify. And if that mock
+         * throws, then boom. No snapshot update and a big old confusing test
+         * failure.
+         */
+        jest.restoreAllMocks();
```

Comment:
Is this something worth adding to the global `afterEach()`?

Reply:
I don't know and I don't want to do that just in case it causes other problems.